Realm/Realm/DatabaseTypes/RealmCollectionBase.cs

```diff
@@ -93,27 +93,45 @@ public abstract class RealmCollectionBase<T>
             }
         }
 
-        [IgnoreDataMember]
+        [IgnoreDataMember, XmlIgnore]
+#if NET6_0
```

Comment:
Shouldn't we use `NET6_0_OR_GREATER`?

Reply:
Oh, yeah, good point. Technically, we only target 6, but it's good to be safe in case we add more tfms on the Realm package.